pulp_rpm/app/viewsets.py

```diff
@@ -1223,6 +1224,11 @@ class RpmAlternateContentSourceViewSet(AlternateContentSourceViewSet, RolesMixin
         ],
     }
 
+    def get_serializer_class(self):
```

Comment:
Instead of overriding the `get_serializer_class()` method you can just use the `@extend_schema(request=RpmAlternateContentSourceRefreshSerializer)` decorator  on the `refresh()` function to specify the serializer.